developer-docs-site/docs/guides/nfts/mint-nft-v2-cli.md

```diff
@@ -0,0 +1,918 @@
+               --profile default                                          \
+               --args                                                     \
+                  string:"Krazy Kangaroos"                                \
+                  string:"https://www.link-to-your-collection-image.com"  \
+                  u64:3                                                   \
+                  u64:5                                                   \
+                  u64:100                                                 \
+                  string:"Krazy Kangaroo #1"                              \
+                  string:"https://www.link-to-your-token-image.com"       
+```
+
+:::info
+When you sign and submit a transaction with an account's private key, you automatically pass the first `&signer` parameter to the function.
+
+Running an entry function with `--profile default` signs and submits the transaction for the `default` profile, which is why you don't need to provide the signer to the `--args` parameter list.
+:::
+
+Once you run that command and confirm that you'd like to submit the transaction, you've created your collection on-chain on the `devnet` network.
+
+This means we can run the `mint` function now:
+
+```shell
+aptos move run --function-id default::create_nft::mint   \
+               --profile default                         \
+               --args address:nft-receiver 
+```
+
+Congratulations! You've created a collection, minted an NFT, and transferred the NFT to another account.
+
+To view the events in this transaction, paste your transaction hash in the Aptos explorer search bar and navigate to the events section, or directly go to:
+
+https://explorer.aptoslabs.com/txn/YOUR_TRANSACTION_HASH_HERE/events?network=devnet
+
+You should see a `0x4::collection::MintEvent` and a `0x1::object::TransferEvent`.
+
+## 2. Automating the mint function with a resource account
+
+The issue with the code we've written so far is that it requires explicit approval from the creator to mint a token. The process isn't automated and the receiver doesn't ever approve of receiving the token.
+
+The first step to improving the flow of this process is automating the creator's approval. We can do this with the use of what's called a resource account.
+
+To achieve this, in this section we'll show you how to:
+
+- Create the NFT collection with a resource account
+- Store the capability to sign things with the resource account, a `SignerCapability`, into the owner's resources on-chain
+- Automate minting the token to the user; that is, write a mint function that works without the collection creator's signature
+
+### What is a resource account?
+
+A resource account is essentially an account that another account can own. They are useful for separating and managing different types of resources, but they're also capable of delegating decisions to sign transactions to the logic in a smart contract.
```

Comment:
This is not the best way to explain resource accounts IMO. Especially for this example, the resource account is actually used as a way to store resources and control (over the collection) that will later be used by the module itself.

Reply:
Hm yea, this is not a particularly good explanation. It's actually pretty confusing in its current form, I'll revise it.

Reply:
always good to provide a link to any dedicated material if we have it.